src/bias/PBMetaD.cpp

```diff
@@ -340,10 +371,20 @@ PBMetaD::PBMetaD(const ActionOptions& ao):
   adaptive_(FlexibleBin::none),
   grid_(false),
   wgridstride_(0),
+  pf_n_(0), do_pf_(false),
   mw_n_(1), mw_dir_(""), mw_id_(0), mw_rstride_(1),
   walkers_mpi_(false), mpi_nw_(0),
   do_select_(false)
 {
+
```

Comment:
I just noticed I accidentally left these comments in from when I was debugging mpi, I will remove them when I do other suggested changes